documentation/src/main/asciidoc/topics/ref_cache_modes_comparison.adoc

```diff
@@ -85,6 +85,14 @@ ifdef::productized[]
 | _Owner nodes_
 | *Complete*
 
+| Scattered
+| *Yes*
+| _Medium_ (primary)
+| _Higher_ (single RPC)
+| Cluster capacity^1^
```

Comment:
this is incorrect. the cluster capacity is divided by 2 and not "num_owners"

Reply:
should be fixed now.